frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -511,12 +511,12 @@ const NodesPage = connect<{}, MapDispatchToProps>(
         isMatch: getNodeRoleMatch,
         items: [
           {
-            id: 'master',
-            title: t('console-app~Master'),
+            id: 'control-plane',
+            title: t('console-app~control-plane'),
```

Comment:
The role label value is `control-plane`, so make the filter reflect the actual value.